2017/07/01_playing_with_time.py

```diff
@@ -72,6 +74,9 @@ def date_to_string(date_or_datetime: Union[date, datetime]) -> str:
     :param date_or_datetime: a date or datetime object to convert
     """
 
+    date_string = date_or_datetime.isoformat()
+
+    return date_string[:10]
```

Comment:
This was before I saw the strptime( ) thing in the docs, but, yeah, that's probably the better way.